project/Publish.scala

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
```

Comment:
in https://github.com/lightbend/akka-commercial-addons/pull/237#issuecomment-345219334 @patriknw suggested we might want to preserve the year range - I don't have a strong preference myself but perhaps there's some reasons to do or avoid that?